### ISSUE TITLE
build issues

### ISSUE THREAD

**Issue by jodonnell** [amasad/debugjs.com#8]
Hey, very cool project.  I wanted to play around with it but I'm having problems installing it.  I did `npm install -g component`.  Then `component build`.  It had a hard time finding your fork of node-util, I have no idea why cause I could see your fork fine from github.  Anyway I just did `git clone git@github.com:amasad/node-util.git components/amasad/node-util` and got to the next problem.

`error : Error: failed to read "./lib/devtools"'s file "scope_template.js"`
i just ran `touch lib/devtools/scope_template.js` to see what would happen and find that lib/devtools/stack_template.js and lib/header/save_template.js are missing as well.

I don't get what I am suppose to do here, an inspection makes me think they are suppose to get generated somehow (maybe through Hogan?) but I'm not too familiar with these front end tools.

Thanks for any help!

**Comment by amasad** [comment]
I'm at an old version of component. That's probably the issue. Maybe they changed the fact that you can require html templates. Use `npm install -g component@0.19.5`. Hope that helps.

**Comment by jodonnell** [comment]
That helped a little bit but I still ran into problems with the build.  I looked at chrisuehlinger's fork and he had to put some dependencies directly into the component.json file.  I did that and had to add a bunch more.  Finally ran into a wall with `error : failed to lookup "visionmedia-debug"'s dependency "guille-ms.js"`.  Seems this project has been renamed rauchg/ms.js.  I think the old version of the dependency is trying to find a file that is no longer on github.  I'll tinker with it a bit more.
